packages/sapience/src/components/forecasting/MarketGroupChart.tsx

```diff
@@ -58,7 +58,20 @@ const MarketGroupChart: React.FC<MarketGroupChartProps> = ({
         typeof point.indexClose === 'number'
           ? point.indexClose / 1e18 // Scale Wei down by 10^18
           : point.indexClose; // Keep null/undefined as is
-      return { ...point, indexClose: scaledIndexClose };
+
+      const scaledMarkets: { [marketId: string]: number | undefined } = {};
+      if (point.markets) {
+        Object.entries(point.markets).forEach(([marketId, value]) => {
+          scaledMarkets[marketId] =
+            typeof value === 'number' ? value / 1e18 : value;
```

Comment:
This case could just be for when it's undefined, might be a clearer way to write this?